src/java.net.http/share/classes/jdk/internal/net/http/HttpClientImpl.java

```diff
@@ -756,6 +1041,48 @@ void wakeupSelector() {
             selector.wakeup();
         }
 
+        void abort(Throwable t) {
+            boolean closed = this.closed;
+            errorRef.compareAndSet(null, t);
+            if (debug.on()) {
+                debug.log("aborting selector manager(closed=%s): " + t, closed);
+            }
+            t = errorRef.get();
+            boolean inSelectorThread = owner.isSelectorThread();
+            if (!inSelectorThread) {
+                // abort anything pending, then close
+                abortPendingRequests(owner, t);
+            }
+            Set<SelectionKey> keys = new HashSet<>();
```

Comment:
I'm not quite understand, what for is this `keys` are here?
Nothing happens with this collection after `addAll` call